### PR TITLE
Task create and populate database and CMS Form

### PR DESCRIPTION
**Pull request by JamesCullen2**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -0,0 +1,57 @@
+<?php
+
+?>
+
```
Comment (kyam-mayden) [inline index.php:4]: HTML opening tags, header, body, doctype etc missing

File: index.php [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+<h1>Portfolio Edit Page</h1>
+    <br><br>
+    <h2>Header Nav Bar Links</h2>
+    <br><br>
+<form method="post" action="admin.php">
+    Type: <select name="type">
```
Comment (kyam-mayden) [inline index.php]: Should probably be using <label> tags in these instances

File: index.php [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    Subtitle: <input type="text" name="subtitle">
+    Description: <input type="text" name="description">
+</form>
+<input type="submit">
+
+<h2>Contact</h2>
+<br><br>
+<form method="post" action="admin.php">
+    Type: <select name="type">
+        <option value="email">Email</option>
```
Comment (kyam-mayden) [inline index.php]: These options could be populated from your database - would be more flexible - can share code with you if wanted

File: index.php [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+        <option value="tel">Tel</option>
+        <option value="social">Social</option>
+    </select>
+    Description: <input type="text" name="description">
+</form>
+<input type="submit">
+
+<h2>Images</h2>
+<br><br>
```
Comment (kyam-mayden) [inline index.php]: (being picky as I have used BR tags too) should probably achieve spacing in CSS file rather than HTML

File: index.php [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+<form method="post" action="admin.php">
+    Subtitle: <input type="text" name="subtitle">
+    Description: <input type="text" name="description">
+</form>
+<input type="submit">
+
```
Comment (kyam-mayden) [inline index.php]: html and body closing tags missing

File: index.php [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+        <option value="about me">About Me</option>
+        <option value="portfolio">Portfolio</option>
+        <option value="home">Contact</option>
+    </select>
+    Link url: <input type="url" name="link url">
+    </form>
+    <input type="submit">
+</form>
+
+<h2>About Me</h2>
```
Comment (kyam-mayden) [inline index.php]: Maybe consider div's (or equivalent) to separate out the page

File: index.php (hunk 2 above)
Comment (kyam-mayden) [reply index.php]: label tags

File: index.php (hunk 1 above)
Comment (JamesCullen2) [reply index.php:4]: done

File: index.php (hunk 2 above)
Comment (JamesCullen2) [reply index.php]: done

**Review by kyam-mayden — CHANGES_REQUESTED** [review]
Some changes requested

File: index.php (hunk 3 above)
Comment (JamesCullen2) [reply index.php]: Will do this in my next task to facilitate linking form with DB once I am clear on exactly what I need to pull

File: index.php (hunk 4 above)
Comment (JamesCullen2) [reply index.php]: ok cool, I'll leave them for now because I don't yet have a CSS file linked !

File: index.php (hunk 5 above)
Comment (JamesCullen2) [reply index.php]: added

File: index.php (hunk 6 above)
Comment (JamesCullen2) [reply index.php]: added

File: index.php [hunk 7]
```diff
@@ -2,56 +2,83 @@
...
+        <form method="post" action="admin.php">
+            <label for="subtitle">Subtitle</label>
+            <input type="text" name="subtitle">
+            <label for="description">Description</label>
+            <input type="text" name="description">
+        </form>
+        <input type="submit">
+    </form>
+    </div>
+
```
Comment (kyam-mayden) [inline index.php:43]: Whitespace

File: index.php [hunk 8]
```diff
@@ -2,56 +2,83 @@
 
 ?>
 
-<h1>Portfolio Edit Page</h1>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>James Cullen Portfolio CMS</title>
+</head>
+
```
Comment (kyam-mayden) [inline index.php:10]: white space

File: index.php [hunk 9]
```diff
@@ -2,56 +2,83 @@
 
 ?>
 
-<h1>Portfolio Edit Page</h1>
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (marty-crane) [inline index.php]: might be worth adding HEADER tag for meta-data

**Review by kyam-mayden — APPROVED** [review]
Looks good

**Pull request merged**